packages/wrangler/src/config/config.ts

```diff
@@ -128,7 +128,17 @@ export interface ConfigFields<Dev extends RawDevConfig> {
 	 */
 	assets:
 		| string
-		| { bucket: string; include: string[]; exclude: string[] }
+		| {
+				bucket: string;
+				include: string[];
+				exclude: string[];
+				cache_control: {
+					browser_TTL: number | undefined;
+					edge_TTL: number | undefined;
```

Comment:
```suggestion
					browser_TTL: number | undefined;
					edge_TTL: number | undefined;
```
These should good defaults for publish